### DISCUSSION TITLE
Onboarding commit message ignores lowercase char

### DISCUSSION THREAD

**Discussion by simone-dandreta** [renovatebot/renovate#22342] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.0.0

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

My default.json file is:
`{
     "extends": ["config:base", ":dependencyDashboard"],
     "enabledManagers": ["dockerfile"],
     "commitMessageAction": "bump",
     "onboardingCommitMessage": "trivial: Configure Renovate"
}`

but when I look at the onboarding commit message I see:
`Trivial: Configure Renovate` with capital 'T'

and this breaks our CI checks on semantic commit messages. 
Am I missing something?

### What you tried so far.

You can look at the default.json file posted above.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
I've confirmed the same behavior manually. Converted this discussion to an issue: #22346

**Reply by simone-dandreta** [reply]
Thanks @rarkins
